.travis.yml

```
@@ -7,6 +7,12 @@ python:
   - "3.7"
   - "pypy3.5"
 
+branches:
+  only:
+  - gonzobot
+  # Allow tagged releases in the format v1.0
+  - /^v\d+\.\d+(\.\d+)?(-\S*)?$/
```

```suggestion
  - /^v\d+\.\d+.*$/
```
This regexp is overcomplex for what it does. Assuming you simply want to confirm that the release matches `v1.0` with anything following it, as this regexp seems to imply.